lib/util/git/index.spec.ts

```diff
@@ -269,6 +276,22 @@ describe('util/git/index', () => {
     it('should return true when custom author is unknown', async () => {
       expect(await git.isBranchModified('renovate/custom_author')).toBeTrue();
     });
+
+    describe('isBranchModified() with non-null cache', () => {
```

Comment:
Please remove this `describe`-block, move `mockReturnValue(false)` statement into each of inner tests. Also please reformulate duplicate test descriptions, so that IDE test runner plugins will have less chance to confuse them.